llvmlite/tests/test_dylib.py

```diff
@@ -0,0 +1,35 @@
+from . import TestCase
+
+from llvmlite import binding as llvm
+from llvmlite.binding import dylib
+import platform
+from ctypes.util import find_library
+import unittest
+
+@unittest.skipUnless(platform.system() in ["Linux", "Darwin"], "Unsupport test for current OS")
```

Comment:
not sure what the error message should look like, okay to just assert that an exception is raised?